File: conditionals/old_enough.py
#Robert Nelson, Old Enough- Python

name = input("What is your name?\n")

if name == "Robert":
    print("Hi Robert!")
else:
    print(f"Hello {name}!")


num = int(input("How old are you?\n"))
#computers read top to bottom, check the least likely first
if num == 0: #<= if always starts the conditional
    print("How can you type at 0?")
elif num >=18: #<= everything in between should be elif
    print("You can vote.")
elif num >=16:
    print("You can drive.")
elif num >=15:
    print("You can get a learners permit.")
elif num >=5:
    print("You can go to school.")
else: #<= else always ends the conditional
    print("You can't vote, drive, get a learners permit, and go to school.")

File: conditionals/old_enough.c
//Robert Nelson, Old Enough - C
#include <stdio.h>
#include <string.h> //needed to compare strings
char name[20];
int num;
int main(void){
    
    printf("Hello, what is your name?\n");
    scanf("%s", name);
    printf("Hello %s, \n", name);
    
    printf("How old are you?\n");
    scanf("%d", &num);
    if(num == 0){
        printf("How can you type at 0?\n");
    }else if(num >= 18){
        printf("You can vote.\n");
    }else if(num >= 16){
        printf("You can drive.\n");
    }else if(num >= 15){
        printf("You can get a learners permit.\n");
    }else if(num >= 5){
        printf("You can go to school.\n");
    }else{
        printf("You can't vote, drive, get a learners permit, and go to school.\n");
    }

    return 0;
}

